2-Browsers/Week1/homework/ex2-aboutMe/index.js

```diff
@@ -9,4 +9,20 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
 3. Look in the css file!
 ------------------------------------------------------------------------------*/
 
-// TODO add your JavaScript code here.
+const liTags = document.querySelectorAll('ul li');
+liTags.forEach((liTag) => {
+  const tagMatch = liTag.childNodes[1];
+  const addRedColor = liTag.classList.add('list-item');
+  if (tagMatch.id === 'nickname') {
```

Comment:
Hey @mysr3809 consider using switch case here:
```suggestion
   switch(tagMatch.id) {
    case 'nickname':
      // code block
      break;
    case 'fav-food':
      // code block
      break;
    default:
      // code block
      break;
  }
```
switch case is faster and easier to read than else if

Reply:
I changed my code to switch case